app/views/staff/events/_proposal_tags.html.haml

```diff
@@ -1,8 +1,8 @@
 #show-proposal-tags
-  - if event.valid_proposal_tags.empty? && current_user.organizer_for_event?(event)
+  - if !event.public_tags? && current_user.organizer_for_event?(event)
```

Comment:
This shouldn't check for public_tags since this looks like the organizer's page for managing them.